_posts/2020-10-16-kaitai-struct-v0.9-released.md

```diff
@@ -13,41 +13,43 @@ categories: news
 After a lot of time and effort, Kaitai project is happy to announce
 release of new major version of Kaitai Struct, declarative markup
 language to describe various binary data structures — binary file
-formats, network stream packets, etc.
+formats, network packets, etc.
 
 The basic idea of Kaitai Struct is that a particular format can be
-described using Kaitai Struct language (in a `.ksy` file), which then
-can be compiled using `kaitai-struct-compiler` into source files in
-one of the supported programming languages. These modules will include
-a generated code for a parser that can read described data structure
-from a file / stream and provide access to its contents in a nice,
+described using Kaitai Struct language (in a `.ksy` file, that stands
+for Kaitai Struct YAML), which then can be compiled using
```

Comment:
The point of this paragraph is to just briefly introduce the idea and principle of Kaitai Struct, and I don't think that the intended audience needs to know the expansion of the `.ksy` abbreviation.

If one decides to get to know KS deeper, it's expanded on every page in the format gallery (e.g. [android_img](https://formats.kaitai.io/android_img/index.html#:~:text=Kaitai%20Struct%20YAML)):

> ## Format specification in Kaitai Struct YAML

Reply:
I mean there is no KSY language. The language js KS and it is defined not in terms of YAML, it can be serialized into JSON (`ksj` extension), "PSON" (python AST for literals, there is no official name, so I have named it so, `ksp` extension), BSON(`ksbb` extension), CBOR (`ksbc` extension), MSGPack (`ksbm` extension) or even a custom binary format (`ksbn` extension, `n` stands for `native`). YAML is only a one of a serialization schemes. So IMHO we should not call the language `KSY`, just `Kaitai Struct` abbreviated as `KS`.

Reply:
> I mean there is no KSY language. The language is KS and (...)

Your comment doesn't fit here, isn't it a response to https://github.com/kaitai-io/kaitai-io.github.io/pull/17#discussion_r507205936 instead?